alerts-ui/src/app/app-navbar/app-navbar.component.html

```diff
@@ -1,6 +1,13 @@
-<nav class="navbar navbar-default">
+<nav class="navbar navbar-default navbar-fixed-top">
   <div class="container-fluid">
     <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse" [attr.aria-expanded]="!collapsed"
+        (click)="toggleCollapse()" [ngClass]="{'collapsed': collapsed}">
+        <span class="sr-only">Toggle navigation</span>
```

Comment:
is this text displayable? if it is, we need to use i18n, the approach would be like `{{{'label.toggleNavigation' | translate}}}`